src/main.js

```diff
@@ -6,23 +6,34 @@ var titleInput = document.querySelector('.title-input');
 var bodyInput = document.querySelector('.body-input');
 var ideaFormSection = document.querySelector('.idea-form');
 
+window.addEventListener('keydown', formValidation);
```

Comment:
We might have to go back and set this specifically to "form" rather than window.